lib/perl/lib/Apache/TS/AdminClient.pm

```diff
@@ -30,28 +30,27 @@ use Apache::TS;
 
 # Mgmt API command constants, should track ts/mgmtapi.h
```

Comment:
I might be missing it, but I don't see a removal for the corresponding line in `ts/mgmtapi.h`

Reply:
Yeh, it's not in ts/mgmtapi.h, it's deep in the core. Look for READ_FILE in there :-)

Reply:
We should probably update or remove that comment then?

Reply:
Yeh, it's not correct. I got fooled by this too initially.